components/DetailPageSidebar/detail-page-sidebar.scss

```diff
@@ -0,0 +1,85 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@include exports("detail-page-sidebar") {
+  .bx--detail-page-sidebar__container {
+    margin: 0 5% 0 0;
+    .details-description,
+    .tag-description {
+      font-size: 0.875rem;
+      word-wrap: break-word;
+    }
+    .tag-description {
+      margin: 0 0 2rem 0;
+      > span {
+        font-weight: 600;
```

Comment:
`font-weight: 600` isn't in our supported ranges; 300 is light, 400 is regular, 700 is bold. Anything else and it's the browser trying to fake a style, versus actually using the font files. Leads to some discrepancies across typography between browsers (Chrome is great at this, Firefox is only okay)